src/App.tsx

```diff
@@ -9,7 +9,9 @@ import { getOverview, getQueuedPlans } from './planactions';
 import { RouteComponentProps } from 'react-router-dom';
 import { IPlan, IPlanObject } from './queueserver';
 import { PlanList } from './plan_list';
+import { HistoricalPlanList } from './historical_plan_list';
```

Comment:
We should consider getting rid of the inconsistency in file naming versus class naming, e.g. make the file `HistoricalPlanList.tsx` too.

Reply:
Updated